Makefile

```diff
@@ -50,30 +52,81 @@ v$(VERSION).tar.$(SRC_EXT):
 $(VERSION).tar.$(SRC_EXT):
 	curl -f -L -O '$(SOURCE)'
 
+$(DEB_BUILD)/$(NAME)-$(VERSION): $(SPEC) $(SOURCES)
+	# Unpack tarball
+	export TAR_OPTIONS="--owner=0 --group=0 --numeric-owner"
+	mkdir -p $(DEB_BUILD)
+	tar -C $(DEB_BUILD) --strip-components=1 --atime-preserve \
+	  -xpf _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+	ln _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT) \
+	  $(DEB_TOP)/$(NAME)-$(DEB_VERS).tar.$(SRC_EXT)
+
+$(DEB_TOP)/.patched: $(DEB_BUILD)/$(NAME)-$(VERSION) $(PATCHES)
+	# extract patches for Debian
+	mkdir -p ${DEB_BUILD}/debian/patches
+	mkdir -p $(DEB_TOP)/patches
+	for f in $(PATCHES); do \
+	  rm -f $(DEB_TOP)/patches/*; \
```

Comment:
Do you want this inside the loop?

Reply:
No, those directories only need to be created once.